sonar/modules/documents/templates/documents/record.html

```diff
@@ -46,7 +46,7 @@ <h5 class="my-4">{{ _('document_type_' + record.documentType) }}</h5>
       {% endif %}
     </div>
     <div class="col">
-      <h1 class="text-primary">{{ title }}</h1>
+      <h1 class="text-primary">{{ title | nl2br | safe }}</h1>
```

Comment:
I think `safe` is not required if you use `nb2br` because the this one does already the safe operation

Reply:
No. It's the case in angular, but not in jinja with this custom template.